### PR TITLE
Update `requirePaddingNewlinesBeforeKeywords` rule

### LINKED ISSUE #4: Fix issue in padding newlines before keywords rule
When using ES6 syntax, the "requirePaddingNewlinesBeforeKeywords" rule fails when exporting a function. For example:

```js
export function create() {
   …
}
```

Will return the error:

```bash
Keyword `function` should have an empty line above it:
    89 | */
    90 |
    91 |export function create() {
--------------^
```

However, if the `export` keyword is removed, the rule works.

### PR DESCRIPTION
**Pull request by ruiquelhas**
Resolves #4.

### CODE REVIEW

**Comment by nunofgs** [comment]
I think we should try to get this fixed in JSCS rather than disable it. Otherwise, if we merge this, then this will be considered valid:

```js
function one() {
}
function two() {
}
```

When, in fact, we want a newline before each `function`.

**Comment by ruiquelhas** [comment]
Actually, in this specific case, I guess because we have `requirePaddingNewLinesAfterBlocks`, that code snippet will be invalid.

File: .jscsrc [hunk 1]
```diff
@@ -46,7 +46,7 @@ requirePaddingNewLinesAfterBlocks: { allExcept: ['inCallExpressions', 'inNewExpr
 requirePaddingNewLinesAfterUseStrict: true
 requirePaddingNewLinesBeforeExport: true
 requirePaddingNewLinesBeforeLineComments: { allExcept: firstAfterCurly }
-requirePaddingNewlinesBeforeKeywords: [ case, do, finally, for, function, if, return, switch, try, typeof, void, while, with ]
+requirePaddingNewlinesBeforeKeywords: [ case, do, finally, for, if, return, switch, try, typeof, void, while, with ]
```
Comment (ruipenso) [inline .jscsrc]: Remove space before and after brackets.

Comment (ruiquelhas) [inline .jscsrc]: We are using this format for other rules also. Either we keep it like this or we change them all. In any case, this is a perfectly valid format according to the latest YAML [spec](http://www.yaml.org/spec/1.2/spec.html).

Comment (ruiquelhas) [inline .jscsrc]: #12 should address this.

**Comment by ruipenso** [comment]
@ruiquelhas Please rebase.

**Pull request merged**